_episodes/05-creating-histograms.md

```diff
@@ -465,7 +427,7 @@ Finally we label our axes and display the histogram, shown here:
 > ## Histograms for the morphometrics challenge (10 min - optional, not included in timing)
 >
 > Using the grayscale and color histogram programs we developed in this episode,
-> create histograms for the bacteria colonies in the
-> **Desktop/workshops/image-processing/10-challenges** directory. Save the
-> histograms for later use.
+> create histograms for the bacteria colonies in the following images: `00-colonies01.jpg`,
+> `00-colonies02.jpg` and `00-colonies03.jpg`. 
+> Save the histograms for later use.
 {: .challenge}
```

Comment:
Should we include a solution for this challenge?

Reply:
I was looking back through the episode - and I actually think perhaps this whole exercise should be removed? In this episode they focus on making masks of specific shapes manually by finding coordinates in the image. For these bacterial colony images, you would instead make these masks via thresholding (especially for images 02 and 03 - where there are so many!). A histogram will have to made for these in the challenges episode anyway, so I'm not sure it also needs to be here. What do you think @uschille, @tobyhodges?